plugins/modules/gitlab_runner.py

```diff
@@ -33,7 +33,8 @@
   - Samy Coenen (@SamyCoenen)
   - Guillaume Martinez (@Lunik)
 requirements:
-  - python-gitlab >= 1.5.0
+  - python-gitlab >= 1.5.0 for legacy runner registration workflow (runner registration token - https://docs.gitlab.com/runner/register/#register-with-a-runner-registration-token-deprecated)
+  - python-gitlab >= 4.0.0 for new runner registration workflow (runner authentication token - https://docs.gitlab.com/runner/register/#register-with-a-runner-authentication-token)
```

Comment:
The lines are too long, and better warp the URL in `U(...)` so it will definitely be treated as an URL:
```suggestion
  - python-gitlab >= 1.5.0 for legacy runner registration workflow
    (runner registration token - U(https://docs.gitlab.com/runner/register/#register-with-a-runner-registration-token-deprecated))
  - python-gitlab >= 4.0.0 for new runner registration workflow
    (runner authentication token - U(https://docs.gitlab.com/runner/register/#register-with-a-runner-authentication-token))
```